tests/test_servefile.py

```diff
@@ -9,6 +10,8 @@
 import tarfile
 import time
 import urllib3
+from datetime import timedelta
+from requests.exceptions import ConnectionError
```

Comment:
fyi: i think these should be sorted according to the `from` word.

and why use `datetime.datetime` but `timedelta` directly? i'd also do `from datetime import datetime`.

Reply:
> fyi: i think these should be sorted according to the `from` word.

Isn't that what I'm doing? I don't get it :smile: 

> and why use `datetime.datetime` but `timedelta` directly? i'd also do `from datetime import datetime`.

(I consider `from datetime import datetime` suboptimal personally, because where `datetime` is used it can easily become unclear if it's the module or the class. I'd rather bring `timedelta` to `datetime.timedelta` if your point is consistency.) I can adjust that to whatever @sebageek wants to have. It would be a waste of time to fix "for you" if he ends up wanting something else.

Reply:
> waste of time to fix "for you"

i understand completely. all my remarks were more about "getting your pr in order as much as you'd want to" so that seba wouldn't have to, in case some of mine and their potential concerns overlapped 🙂.

Reply:
I tend to nowadays sort my imports in three blocks (standard library, external imports, internal imports) and then alphabetically, though looking at the code that was already there this was not the case when I originally wrote that code. But I'll have a go at that when I again try to reformat the whole codebase, that is still using tabs...